compose.yml

```diff
@@ -86,7 +86,7 @@ services:
     develop:
       watch:
         - path: backend/bin
-          action: rebuild
```

Comment:
The Docker docs say that `rebuild` is the same as `docker compose up --build` but the latter command works and this rebuild does not.
It results in saying the application could not be restarted

However, `sync+restart` seems to be doing the trick since we sync the binary and restart the process.